ubuntu-mainline-kernel.sh

```diff
@@ -369,6 +369,22 @@ latest_local_version() {
 }
 
 remote_html_cache=""
+version_parse() {
```

Comment:
Could we maybe rename this method to eg _parse_remote_versions_ to make it clear that its meant to be used just for sorting remote versions only (and not local versions)?

Reply:
That makes sense